AllReadyApp/Web-App/AllReady.UnitTest/Notifications/GetUserUnenrollsNotificationModel.cs

```diff
@@ -0,0 +1,139 @@
+
+            _firePrev = new Campaign()
+            {
+                Name = "Neighborhood Fire Prevention Days",
+                ManagingOrganization = _htb,
+                CampaignContacts = new List<CampaignContact>()
+            };
+            _firePrev.CampaignContacts.Add(new CampaignContact
+            {
+                Campaign = _firePrev,
+                Contact = _contact1,
+                ContactType = (int)ContactTypes.Primary
+            });
+            _htb.Campaigns.Add(_firePrev);
+            _queenAnne = new Activity()
+            {
+                Id = 1,
+                Name = "Queen Anne Fire Prevention Day",
+                Campaign = _firePrev,
+                CampaignId = _firePrev.Id,
+                StartDateTime = new DateTime(2015, 7, 4, 10, 0, 0).ToUniversalTime(),
+                EndDateTime = new DateTime(2015, 12, 31, 15, 0, 0).ToUniversalTime(),
+                Location = new Location { Id = 1 },
+                RequiredSkills = new List<ActivitySkill>(),
+                Tasks = new List<AllReadyTask>()
+            };
+            _queenAnne.UsersSignedUp = new List<ActivitySignup>
+            {
+                new ActivitySignup
+                {
+                    Id = 1,
+                    PreferredEmail = "testuser@gmail.com",
+                    PreferredPhoneNumber = "(555)555-1234",
+                    User = _user1
+                }
+            };
+            _task1 = new AllReadyTask()
+            {
+               Id = 1,
+               Activity = _queenAnne,
+               Name = "Task 1",
+                StartDateTime = new DateTime(2015, 7, 4, 10, 0, 0).ToUniversalTime(),
+                EndDateTime = new DateTime(2015, 12, 31, 15, 0, 0).ToUniversalTime(),
+                Organization = _htb
+            };
+            _task1.AssignedVolunteers = new List<TaskSignup>()
+            {
+                new TaskSignup
+                {
+                    Id = 1,
+                    User = _user1,
+                    Task = _task1,
+                    Status = "Assigned",
+                    StatusDateTimeUtc = new DateTime(2015, 7, 4, 10, 0, 0).ToUniversalTime()
+                }
+            };
+            _queenAnne.Tasks.Add(_task1);
+            context.Users.Add(_user1);
+            context.Contacts.Add(_contact1);
+            context.Organizations.Add(_htb);
+            context.Activities.Add(_queenAnne);
+            context.SaveChanges();
+        }
+
+        [Fact]
+        public void ModelCanBeCreatedFomExistingActivity()
+        {
+            var context = ServiceProvider.GetService<AllReadyContext>();
+            var query = new UserUnenrollsNotificationQuery { ActivityId = 1 };
+            var handler = new UserUnenrollsNotificationQueryHandler(context);
+            var result = handler.Handle(query);
+            Assert.NotNull(result);
+            Assert.True(_queenAnne.Id == result.ActivityId, "ActivityId does not match");
```

Comment:
I'm not sure we need _all_ these asserts, as at this point the properties aren't really transformed, they are just returned from the in-mem store. How about the null check, and that the collections contain the correct count? Otherwise, we're just really testing language and library features (like object initialization and the in-mem store).